developer_productivity_analytics_github/infra/airbyte/main.tf

```diff
@@ -0,0 +1,89 @@
+
+// source
+resource "airbyte_source_github" "my_source_github" {
+  configuration = {
+    credentials = {
+      source_github_authentication_personal_access_token = {
+        personal_access_token = var.personal_access_token
+      }
+    }
+    repository  = "bishalbera/Skillify"
```

Comment:
Maybe you can add here a placeholder repository name, and also below, instead of using your real name.

Reply:
ok fixing this